common/xfer/plugin_spec.go

```diff
@@ -7,8 +7,8 @@ import (
 	"sort"
 
 	"github.com/davecgh/go-spew/spew"
-	"github.com/mndrix/ps"
 	"github.com/ugorji/go/codec"
+	"github.com/weaveworks/ps"
```

Comment:
gvt works nicely for this.

We did this multiple times for the docker client library for instance.  It was always out intention to upstream the changes though, which may be different here?  Have you tried asking the original author?